src/app/components/buscadorTurnosPrestaciones/turnos-prestaciones.html

```diff
@@ -51,38 +54,37 @@
-                            <plex-bool label="Fecha" [(ngModel)]="columnas.fecha">
-                            </plex-bool>
-                            <plex-bool label="Documento" [(ngModel)]="columnas.documento">
-                            </plex-bool>
-                            <plex-bool label="Paciente" [(ngModel)]="columnas.paciente">
-                            </plex-bool>
-                            <plex-bool label="Prestación" [(ngModel)]="columnas.tipoPrestacion">
-                            </plex-bool>
-                            <plex-bool label="Equipo de Salud" [(ngModel)]="columnas.equipoSalud">
-                            </plex-bool>
-                            <plex-bool label="Estado" [(ngModel)]="columnas.estado">
-                            </plex-bool>
-                            <plex-bool label="Financiador" [(ngModel)]="columnas.financiador">
-                            </plex-bool>
-                            <plex-bool label="Ámbito" [(ngModel)]="columnas.ambito">
-                            </plex-bool>
-                        </plex-grid>
-                    </plex-dropdown>
-                </plex-title>
-                <ng-container *ngIf="selectPrestaciones$ | async as selectPrestaciones">
+        <ng-container *ngIf="state$ | async as state">
+            <plex-title titulo="Listado" size="sm">
+                <plex-button label="Exportar" size="sm" type="success" class="mr-2" (click)="exportPrestaciones()"
+                             [disabled]="!state.enableExport">
+                </plex-button>
+                <plex-dropdown icon="format-list-checks" type="info" size="sm" class="d-inline-block" [right]="true">
+                    <plex-grid cols="3">
+                        <plex-bool label="Fecha" [(ngModel)]="columnas.fecha">
+                        </plex-bool>
+                        <plex-bool label="Documento" [(ngModel)]="columnas.documento">
+                        </plex-bool>
+                        <plex-bool label="Paciente" [(ngModel)]="columnas.paciente">
+                        </plex-bool>
+                        <plex-bool label="Prestación" [(ngModel)]="columnas.tipoPrestacion">
+                        </plex-bool>
+                        <plex-bool label="Equipo de Salud" [(ngModel)]="columnas.equipoSalud">
+                        </plex-bool>
+                        <plex-bool label="Estado" [(ngModel)]="columnas.estado">
+                        </plex-bool>
+                        <plex-bool label="Financiador" [(ngModel)]="columnas.financiador">
+                        </plex-bool>
+                        <plex-bool label="Ámbito" [(ngModel)]="columnas.ambito">
+                        </plex-bool>
+                    </plex-grid>
+                </plex-dropdown>
+            </plex-title>
+            <ng-container *ngIf="selectPrestaciones$ | async as selectPrestaciones">
+                <cdk-virtual-scroll-viewport [itemSize]="60">
```

Comment:
Es importante setear la altura correcta del item para que ande mejor el virtual scroll. Sino se aprecia que queda un vacío abajo. 

```suggestion
                <cdk-virtual-scroll-viewport [itemSize]="50">
```

Reply:
Hecho, es verdad no me habia dado cuenta